app/js/controller/list_controller.js

```diff
@@ -53,6 +53,12 @@ export default class ListController extends Controller {
   }
 
   launchApp(data) {
+    if (data.name == 'Phone') {
```

Comment:
I'm not sure about keying off the `name` here for launching via entry point. Perhaps we should just set an `entryPoint` property in the `data` object that gets passed in here that specifies an optional value. If `data.entryPoint` exists, then we use it.